apps/pipelines/nodes/nodes.py

```diff
@@ -150,12 +159,41 @@ def process_conditional(self, state: PipelineState) -> Literal["true", "false"]:
             return "true"
         return "false"
 
-    @classmethod
-    def get_output_map(cls):
-        """A mapping from the output handles on the frontent to the return values of process_conditional"""
+    def get_output_map(self):
+        """A mapping from the output handles on the frontend to the return values of process_conditional"""
         return {"output_true": "true", "output_false": "false"}
 
 
+class RouterNode(Passthrough, LLMResponseMixin):
+    class Config:
+        extra = "allow"
+
+    __human_name__ = "Router"
+    llm_provider_id: LlmProviderId
+    llm_model: LlmModel
+    prompt: str = "You are an extremely helpful router {input}"
+    num_outputs: NumOutputs = 2
+
+    def process_conditional(self, state: PipelineState):
+        prompt = PromptTemplate.from_template(template=self.prompt)
+        chain = prompt | self.get_chat_model()
+        result = chain.invoke(state["messages"][-1], config=self._config)
```

Comment:
could be useful to use langchains structured output feature: https://python.langchain.com/v0.1/docs/modules/model_io/chat/structured_output/